src/extensionVariables.ts

```diff
@@ -18,6 +18,7 @@ export namespace ext {
     export let reporter: ITelemetryReporter;
     export let tree: AzureTreeDataProvider;
     export let treeView: TreeView<AzureTreeItem<ISubscriptionRoot>>;
+    export let emulatorPassword = 'C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==';
```

Comment:
I think this fits better in `constants.ts` than `extensionVariables.ts`

For the record I feel the same way about settingsKeys below, but that's unrelated to your PR so no need to change if you don't want to.